packages/eslint-config-nimble/package.json

```
@@ -32,7 +32,7 @@
     "lib": "lib"
   },
   "engines": {
-    "node": "^14.18.2 || ^16 || ^17"
+    "node": "^14.18.2 || ^16 || ^17 || ^18"
```

Do we really need to keep all of these versions? How about removing the oldest one:

```suggestion
    "node": "^16 || ^17 || ^18"
```

---

Updated in [0130d9b](https://github.com/nimblehq/eslint-config-nimble/pull/46/commits/0130d9b8c3546a874151989a2073a1b529077780)